src/scripts/main.js

```diff
@@ -1,8 +1,23 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard').children[0];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.map(person => {
```

Comment:
good case to use destructure

```suggestion
people.map({ name, sex, ..... } => {
```

Reply:
done, but I had some problems with 'name', so i commited with --no-verify